LEAF_Request_Portal/form.php

```diff
@@ -3009,6 +3009,20 @@ public function query($inQuery)
             $joins .= "LEFT JOIN (SELECT userName, lastName, firstName FROM {$this->oc_dbName}.employee) lj_OCinitiatorNames ON records.userID = lj_OCinitiatorNames.userName ";
         }
 
+        if(isset($_GET['debugQuery'])) {
+            $debugQuery = str_replace(["\r", "\n"], ' ', 'SELECT * FROM records ' . $joins . 'WHERE ' . $conditions . $sort . $limit);
+            $debugVars = array_map(fn($val):string => is_numeric($val) ? $val : '"'.$val.'"', $vars);
```

Comment:
This ternary operator seems backwards. If you want to return a string, shouldn't it convert to string if it's numeric and leave it alone otherwise?

Reply:
The purpose of line 3014 is to provide a more realistic looking query by wrapping strings in quotes, since SQL expects strings to be quoted while numeric values aren't.

While looking at this, additional logic needs to be added to wrap numbers in quotes in relation to the "data" table, since data is being stored as strings.